### ISSUE TITLE
How to load the training weights of the student model before distillation？

### ISSUE THREAD

**Issue by Ye1133Zi** [open-mmlab/mmrazor#613]
### Checklist

- I have searched related issues but cannot get the expected help.
- I have read related documents and don't know what to do.

### Describe the question you meet

In distillation, I don't know how to modify the distillation configuration file to load my own weights from the last training or other trained model weights

### Post related information

1. The output of `pip list | grep "mmcv\|mmrazor\|^torch"`
   \[here\]
2. Your config file if you modified it or created a new one.

```python
mmrazor/configs/distill/mmcls/kd/kd_logits_resnet34_resnet18_8xb32_in1k.py
```

4. Your train log file if you meet the problem during training.
   I tried adding **"load_from = **'trained_models/resnet18-f37072fd.pth'** , and in the training log it shows that the checkpoint has been loaded. But during training, the accuracy still starts from zero.
![2023-12-04_232301](https://github.com/open-mmlab/mmrazor/assets/92502198/9d2a9eef-aa2c-43de-8a3f-07087158feee)

5. Other code you modified in the `mmrazor` folder.

**Comment by vbercy** [comment]
Hi, I've got the same question. I guess we can't resume the training from a simple checkpoint file of the student, as the checkpoint files generated during a MMRazor training are heavier (they may contain the teacher and the student)...

As we have this in the config file for the teacher:
```
    teacher=dict(cfg_path=<teacher_cfg_path>, pretrained=False),
    teacher_ckpt=<teacher_ckpt_path>
```
Have you tried the same structure for the student ?
```
    architecture=dict(cfg_path=<student_cfg_path>, pretrained=False),
    student_ckpt=<student_ckpt_path>
```

**Comment by babajakpum** [comment]
Have you solved this problem?
